src/components/ShoeGrid/ShoeGrid.jsx

```diff
@@ -8,12 +8,22 @@ const ShoeGrid = () => {
   return (
     <Wrapper>
       {SHOES.map((shoe) => (
-        <ShoeCard key={shoe.slug} {...shoe} />
+        <ShoeWrapper>
+          <ShoeCard key={shoe.slug} {...shoe} />
+        </ShoeWrapper>
       ))}
     </Wrapper>
   );
 };
 
-const Wrapper = styled.div``;
+const Wrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  gap: 32px;
+`;
+
+const ShoeWrapper = styled.div`
+  flex: 1 1 275px;
```

Comment:
Done at this level because `ShoeCard > a` exists and overrides any descendant of it with `display: inline`